lib/components/PlaylistPage.js

```diff
@@ -1,15 +1,84 @@
 import React from 'react';
+import Request from 'superagent';
+
 import SearchBar from './SearchBar';
+import SearchResult from './SearchResult';
+import SearchResults from './SearchResults';
 
 const INPUT_KLASS = "spotify-search"
 const INPUT_PLACEHOLDER = "Search Spotify"
 
 export default React.createClass({
+  getInitialState() {
+    return {
+      playlistTracks : {},
+      searchResults : [],
+    }
+  },
+
+  handleSearch(e) {
+    e.preventDefault();
+    let queryValue = e.target.value;
+    this.searchSpotify(queryValue);
+
+    // this.state.searchResults.forEach(function(result){
+    //   console.log("1", result);
+    // });
+  },
+
+  storeResults(tracks) {
+    debugger;
+    self.state.searchResults = tracks;
+    self.setState({ searchResults : self.state.searchResults });
```

Comment:
Good catch, definitely should have used `this` there. In the function making the AJAX call, I set `self` equal to `this` as `this` isn't scoped properly inside the AJAX callback.

I fixed that in my code and still have the same issue. I'll push up my changes.